FogCreek/Glitch-Community
# Wakeful scent: fix bugs related to project deletion

## Links
* https://wakeful-scent.glitch.me/
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/81

## Changes:
* when a user deletes a project from the collection page, reset what projects are on that page (right now there's a weird blank area that's still clickable to the deleted project, and the other projects don't slide back to replace)
* remove the ability to delete a project from the homepage, right now it doesn't work, and seems sort of odd to me that we're encouraging folks to remove their apps from our curated collections? if we want this behavior tho, we'll want to rethink how we pass around deleteItem, so that the collection adjusts what projects it has in state. 

## How To Test:
* add a project to a collection page and try to delete it from that collection page
* add a test collection with a project you own to the list of curated collections that show on the home page, see that you can no longer delete it

## Feedback I'm looking for:
- any bugs this could cause that I'm not thinking of?
- should you actually be able to delete a project from the homepage?
